test/inputStabilityCheck.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { createSelector, setInputStabilityCheckEnabled } from 'reselect'
+import { shallowEqual } from 'react-redux'
```

Comment:
:Hand: strictly speaking we could do this with the `createCacheKeyComparator()` util.  And for that matter, given that we've got it being called in one or two places, maybe we ought to just extract `const shallowEqual = createCacheKeyComparator(defaultEqualityCheck)` into a `utils.ts` file and reuse that.

Reply:
createCacheKeyComparator assumes an array, which { a, b } is not 🤔

Reply:
Ah, so it does. Feels annoying pulling in React-Redux just for a shallow equality function, but I guess it's dev-only.

Reply:
yeah, that was my rationale - it's just for tests, doesn't affect the library user